charts/connect/values.yaml

```diff
@@ -2,81 +2,243 @@
 # that every value "{{ .Between.Curly.Braces }}" in this file needs to be
 # explicitly interpolated on the template side by using the `tpl` function.
 
+# This section of values is for 1Password Connect API and Sync Configuration
 connect:
-  applicationName: onepassword-connect
+  # The 1Password Connect API Specific Values
   api:
     name: connect-api
+    # The 1Password Connect API repository
     imageRepository: 1password/connect-api
     resources: {}
     httpPort: 8080
     httpsPort: 8443
+
+  # The 1Password Connect Sync Specific Values
   sync:
     name: connect-sync
     imageRepository: 1password/connect-sync
     resources: {}
     httpPort: 8081
+
+  # The name of 1Password Connect Application
+  applicationName: onepassword-connect
+
+  # The type of Service resource to create for the Connect API and sync services.
+  # See: https://kubernetes.io/docs/concepts/services-networking/service
+  # This by default is NodePort and can also be defined as LoadBalancer.
+  # If serviceType is LoadBalancer then loadBalancerSourceRanges and loadBalancerIP should be defined.
+  # See: https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer
   serviceType: NodePort
+
+  #loadBalancerSourceRanges:
```

Comment:
Just not to forget that there is a lint error to fix

Reply:
This should show green now when the CI is approved.